server/src/grpc/storage_service/mod.rs

```diff
@@ -489,8 +506,9 @@ fn build_schema_from_metric(schema_config: &SchemaConfig, metric: &WriteMetric)
 
     ensure!(
         !write_entries.is_empty(),
-        InvalidArgument {
-            msg: format!("Empty write entires to write table:{}", table_name),
+        ErrNoCause {
+            code: StatusCode::BAD_REQUEST,
+            msg: format!("empty write entires to write table:{}", table_name),
```

Comment:
empty -> Empty? It make me confused, the first character lowercase or uppercase?

Reply:
The style seems to keep the first character lowercase.